integration_tests/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
(Sorry I put a comment here, but I can't comment on `__init__.py` in the root directory.)

The file `/__init__.py` is rather ugly. Do you really need it? Can't you instead set the `PYTHONPATH` to point to the `integration_tests` subdirectory?

Reply:
Also, do we need to add the license header to empty files?

Reply:
I think that would be possible, but it would require to remove "integration_tests." from imports. This feels wrong to me, it's contrary to the "all imports/includes should be rooted at the root" mantra of e.g. Bazel.

Not sure about license headers and empty files. Tensorflow and GRPC do it, protobuf doesn't. I'll add them, seems like we'd be on the safe side this way.